__tests__/integration/mirador/index.html

```diff
@@ -8,7 +8,7 @@
   </head>
   <body>
     <div id="mirador"></div>
-    <script>document.write("<script type='text/javascript' src='../../../dist/mirador.min.js?v=" + Date.now() + "'><\/script>");</script>
+    <script>document.write("<script type='text/javascript' src='../../../dist/mirador3.js?v=" + Date.now() + "'><\/script>");</script>
```

Comment:
We need to keep the name of the package free from version information in order to do releases. "3" just happens to be the semver landmark we've reached, but henceforth we want to be able to easily update to 4, 5, 6, without necessarily making these a part of the name.

https://semver.org/

Reply:
https://github.com/ProjectMirador/mirador/blob/e469d5063f6c77b807a6af3f2b2e468bee5c6a1b/webpack.config.js#L8-L9
OK?